src/js/reducers/signals.test.js

```diff
@@ -249,6 +249,22 @@ describe('signals reducer', function() {
           _idx: 1
         }
       });
+    });
+
+    it('initializes the scene width & height signals set to custom values', function() {
+      var result = signalsReducer(initialState, createScene({width:400, height:100}));
```

Comment:
nit but this'd be more readable if the width and height were indented out onto their own lines